### PR TITLE
refactor: use styled-components for the styling

### PR DESCRIPTION
**Pull request by juniusfree**
Demo: https://www.loom.com/share/d4987ad1ab954adeb70e741399dedb57

### CODE REVIEW

**Review by terryli0095 — APPROVED** [review]

File: wondrous-app/components/Common/TaskSubtaskList/styles.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Task } from '../Task';
 import { TaskInner, TaskWrapper } from '../Task/styles';
 
 // https://www.designcise.com/web/tutorial/why-is-styled-components-styled-wrapper-not-working-with-existing-react-component
-export const SubtaskTaskItem = styled(Task)`
+export const SubtaskTaskItem = styled((props) => <Task {...props} />)`
```
Comment (terryli0095) [inline wondrous-app/components/Common/TaskSubtaskList/styles.tsx:6]: ah nice

Comment (terryli0095) [reply wondrous-app/components/Common/TaskSubtaskList/styles.tsx:6]: @andywong418

Comment (andywong418) [reply wondrous-app/components/Common/TaskSubtaskList/styles.tsx:6]: Makes sense to me!

File: wondrous-app/components/Dashboard/boards/index.tsx [hunk 2]
```diff
@@ -266,22 +266,6 @@ const useGetUserTaskBoard = ({
   };
 };
 
-const useGetPerStatusTaskCountForUserBoard = (loggedInUser) => {
```
Comment (terryli0095) [inline wondrous-app/components/Dashboard/boards/index.tsx:269]: haha nice no idea why it's this way

**Review by terryli0095 — APPROVED** [review]

**Pull request merged**